node/node.go

```diff
@@ -606,7 +607,7 @@ func (n *Node) loadSecurityConfig(ctx context.Context) (*ca.SecurityConfig, erro
 		// - We wait for CreateSecurityConfig to finish since we need a certificate to operate.
 
 		// Attempt to load certificate from disk
-		securityConfig, err = ca.LoadSecurityConfig(ctx, rootCA, krw)
+		securityConfig, err = ca.LoadSecurityConfig(ctx, rootCA, krw, n.config.ForceNewCluster)
```

Comment:
There doesn't seem to be a test case for this, but this second call `to LoadSecurityConfig` may be for the case where maybe the root CA file is deleted (but the TLS keys are on disk), and a join address is provided, so we can re-download the root CA file.

Question: 
1.  Should we actually just download the root CA file to recover, or should this configuration just be an error?
1.  What should happen if the root CA is on disk but corrupt?  Should it also try to generate/download a new CA and download/load the TLS certs?